extensions/variants_test.go

```diff
@@ -18,53 +18,147 @@ import (
 
 func TestEvaluateTypeExpression(t *testing.T) {
 	var (
-		i64Null, _    = parser.ParseType("i64?")
-		i64NonNull, _ = parser.ParseType("i64")
-		strNull, _    = parser.ParseType("string?")
+		i64Null, _      = parser.ParseType("i64?")
+		i64TypeReq      = &types.Int64Type{Nullability: types.NullabilityRequired}
+		i64NonNull, _   = parser.ParseType("i64")
```

Comment:
it's non intuitive how are those different. perhaps a comment should help

Reply:
Done; sorted this list so that function argument definition types are group together, and underlying types are below.